vision/automl/src/test/java/com/google/cloud/vision/samples/automl/DatasetApiIT.java

```diff
@@ -60,29 +60,29 @@ public void testCreateImportDeleteDataset() {
         String.format("test_%s", UUID.randomUUID().toString().replace("-", "_").substring(0, 26));
     // Act
     DatasetApi.createDataset(PROJECT_ID, COMPUTE_REGION, datasetName, false);
-
-    // Assert
-    String got = bout.toString();
+    //
+    // // Assert
+    // String got = bout.toString();
     datasetId =
         bout.toString()
             .split("\n")[0]
             .split("/")[(bout.toString().split("\n")[0]).split("/").length - 1];
-    assertThat(got).contains("Dataset id:");
+    // assertThat(got).contains("Dataset id:");
```

Comment:
nor this

Reply:
sorry - i made an update PR to remove these debugging things